foss4g2023.html

```diff
@@ -790,6 +790,16 @@ <h3>Turning git commit entries into a release page</h3>
     <p>by Vaclav Petras (procedure support by Markus Neteler)</p>
 </section>
 
+<section>
+    <h2>New docker hub organization</h2>
+    <br>
+    <h3>mundialis -> OSGeo</h3>
+    <ul>
+        <li>The GRASS GIS docker images are currently being moved from "mundialis" to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
+    </ul>
+    <img src="img/grass-dockerhub.png" alt="GRASS GIS on Docker Hub" class="stretch">
```

Comment:
I suggest to add (next to this screenshot) also a screenshot of <https://hub.docker.com/r/osgeo/grass-gis>. And, unfortunately, I have no idea how to add a new file to a PR as part of the review process.

Reply:
That has become very easy, using the "Code" button on the right top side of the main PRtab: you can edit the PR by checking it out like this:

![image](https://github.com/ctu-geoforall-lab/grass-gis-talk-foss4g-2023/assets/1295172/a16e82dc-4a7f-4c4f-9760-1c3501f3bc12)

So:
`gh pr checkout 13`

Then edit/add/commit. Then

`git push`

and the modification is  in this PR.